packages/vm/lib/state/stateManager.ts

```diff
@@ -71,7 +71,7 @@ export default class DefaultStateManager implements StateManager {
    * @param address - Address of the `account` to get
    */
   async getAccount(address: Buffer): Promise<Account> {
-    const account = await this._cache.getOrLoad(address)
+    const account = (await this._cache.getOrLoad(address)) as Account
```

Comment:
this is cast with `as Account` because `getOrLoad` was updated to return with optional undefined (`Promise<Account | undefined>`), so it may be better to update the return signature of the function here instead of casting.